backend/src/common/database/types/database.type.ts

```diff
@@ -1,10 +1,11 @@
 import { type Knex } from 'knex';
+import { type ValueOf } from 'shared';
```

Comment:
It's sould re-exported somewhere, likely in common/types/types.ts. Import it from there.